### PR TITLE
[docs] describe new properties of testRunInfo

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
#2216

### CODE REVIEW

**Comment by MargaritaLoseva** [comment]
@dirk-pieterse Please review only changed text, not the whole files.

File: docs/articles/documentation/extending-testcafe/reporter-plugin/reporter-methods.md [hunk 1]
```diff
@@ -124,6 +119,40 @@ reportTestDone (name, testRunInfo, meta) {
...
+
+### screenshots Object
+
+The `screenshot` object provides information about the screenshot captured during the test run. The object has the following properties:
+
+Property            | Type             | Description
+------------------- | ---------------- | --------------------------------------------------------
+`screenshotPath`    | String           | The directory path where the screenshot was saved to.
+`thumbnailPath`     | String           | The directory path where the screenshot's thumbnail was saved to.
+`userAgent`         | String           | The browser where the screenshot was captured.
```
Comment (AndreyBelym) [inline docs/articles/documentation/extending-testcafe/reporter-plugin/reporter-methods.md]: The user agent string of the browser where screenshot was captured.

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 2]
```diff
@@ -288,12 +288,7 @@ testcafe all tests/sample-fixture.js -S -s screenshots
 
 ### -q, --quarantine-mode
 
-Enables the quarantine mode for tests that fail.
-In this mode, a failed test is executed several times.
-The test result depends on the outcome (*passed* or *failed*) that occurs most often.
-That is, if the test fails on most attempts, the result is *failed*.
-
-If the test result differs between test runs, the test is marked as unstable.
+Enables the [quarantine mode](programming-interface/runner.md#quarantine-mode) for tests that fail.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: See [Concurrent Test Execution](common-concepts/concurrent-test-execution.md) to learn more about concurrent test execution. - See [Concurrent Test Execution](common-concepts/concurrent-test-execution.md) for more information about concurrent test execution.

If this option is enabled, TestCafe pauses the test at the moment it fails -> If this option is enabled, TestCafe pauses the test **when** it fails

Specifies the time (in milliseconds) within which selectors attempts  -> attempt (selectors - plural)

Specifies the time (in milliseconds) within which TestCafe makes attempts to successfully execute an assertion if a selector property or a client function was passed as an actual value. -> Specifies the time (in milliseconds) TestCafe attempts to successfully execute an assertion if a selector property or a client function was passed as an actual value.

TestCafe **will** correspond these symbols to any number of characters in the URL. Wildcards at the beginning and end of the rules can be omitted (*.mycompany.com and .mycompany.com have the same effect). -> No future tense. 'correspond these symbols' How do you correspond something? Something corresponds to/with.

Enables colors on the command line -> in the command line.

File: docs/articles/documentation/using-testcafe/programming-interface/runner.md [hunk 3]
```diff
@@ -462,9 +462,7 @@ You can also cancel all pending tasks at once using the [runner.stop](#stop) fun
 The quarantine mode is designed to isolate *non-deterministic* tests (that is, tests that sometimes pass and fail without a clear reason)
 from the rest of the test base (*healthy* tests).
 
-When quarantine mode is enabled, tests are not marked as *failed* after the first unsuccessful run but rather sent to quarantine.
-After that, these tests are run again several times. The outcome of the most runs (*passed* or *failed*) is recorded as the test result.
-A test is separately marked *unstable* if the outcome varies between runs. The test counts the run that was quarantined.
+In this mode, a failed test is executed several times. The test result depends on the outcome (passed or failed) that occurs most often. That is, if the test fails on most attempts, the result is failed. If the test result differs between test runs, the test is marked as unstable.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/programming-interface/runner.md]: Tests are run concurrently against this pool, that is, each test is run in the first free instance. -> first **available** instance

The quarantine mode is designed to isolate non-deterministic tests (that is, tests that sometimes pass and fail without **any apparent** reason) from the rest of the test base (healthy tests).

See Martin Fowler's Eradicating Non-Determinism in Tests article for more information about non-deterministic tests,.

**Pull request merged**